COMMUNITY.md

```diff
@@ -0,0 +1,40 @@
+# Engaging in the radare2 community
+
+
+## Motivation
+It has turned out that relying on people's common sense is actually not
+sufficient for larger software projects, as there are many different
+interpretation of what this means.
+
+So here is a rather vague explaination of what we'd like to see in the r2
+community (gh issues, source code, irc, telegram, discord, conferences)
+
+## What we don't want to see
+We don't want any form of fascism, racism, sexism, homophobia, transphobia,
+ableism, antisemitism and xenophobia in this community.
+We understand, that people make mistakes, and sometimes don't even know what
+they're saying/doing. This is no excuse for intentional harassment,
+discriminatory and abusive behaviour.
+
+## What we want to see
+We wish to have a supportive community of kind people vibing together,
+connected by common interests, which is radare2 and
+software reverse engineering in general.
+
+This does not limit the conversations in our
```

Comment:
no need to specify the protocols. imho any communication in any possible protocol must be tied to it. so just remove the protocols or provide links to the chats below

Reply:
> So here is a rather vague explaination of what we'd like to see in the r2 community (gh issues, source code, irc, telegram, discord, conferences)

So here is a view guidelines of what behavior we expect to have from our r2 community members, during our interaction with others in our source code development, documentation handling, dealing with bug/issues, support and communication with people and our events.